src/comps/Status.tsx

```diff
@@ -69,107 +69,102 @@ const Status = () => {
     useState<EmilyDepositResponse | null>();
 
   // get the query params from the url
-  const router = useRouter();
   const searchParams = useSearchParams();
 
-  const emilyUrl = useAtomValue(emilyUrlAtom);
-  const bitcoinDaemonUrl = useAtomValue(bitcoinDaemonUrlAtom);
+  const { EMILY_URL: emilyUrl } = useAtomValue(bridgeConfigAtom);
+  const handleFetchFromEmily = useCallback(
+    async (txId: string, vout: number) => {
+      try {
+        // create a get request to emily to get the tx status
+        // call /deposit with body { bitcoinTxid: string, vout: number, url: string }
 
-  useEffect(() => {
-    // get the txId from the query params
+        // make emily post request
+        const emilyGetPayload = {
+          bitcoinTxid: txId,
+          vout: vout,
+          url: emilyUrl,
```

Comment:
Do we need to pass emily url here? vs getting it from env in the backend

Reply:
smart!